app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -140,6 +140,47 @@ class MEJSPlayer {
     }
   }
 
+  /**
+   * Handle Mediaelement's 'ended' event
+   * @function handleEnded
+   * @return {void}
+   */
+  handleEnded() {
+    const t = this;
+
+    // No sections content on this page, go no further
+    if (!t.hasSections()) {
+      return;
+    }
+
+    const $sections = $('#accordion').find('.panel-heading[data-section-id]');
+    const sectionsIdArray = $sections.map((index, item) =>
+      $(item).data('sectionId')
+    );
+    const currentIdIndex = [...sectionsIdArray].indexOf(t.currentStreamInfo.id);
```

Comment:
I'm curious about this. Why not just do sectionsIdArray.indexOf?

Reply:
Ah, good catch it's not necessary.  Originally I was calling `indexOf` on an array of javascript elements, or jQuery objects or something, so was using the es6 spread operator `[...myArray]` to coerce the returned "JS node-y" array to a standard array you can call `Array` methods on.   I just didn't take the spread operator out.  I'll update that in a future commit.